src/api/map/routes/map.ts

```diff
@@ -0,0 +1,34 @@
+import { procedure, router } from "../../trpc";
+import { authUser } from "../../auth";
+
+const get = procedure
+  .use(authUser("Mentee"||"Mentor"))
```

Comment:
no need for "Mentee"||"Mentor". every logged-in user should be able to use the map.